public_html/wp-content/plugins/wordcamp-speaker-feedback/wordcamp-speaker-feedback.php

```diff
@@ -16,13 +16,92 @@
 
 define( __NAMESPACE__ . '\PLUGIN_DIR', \plugin_dir_path( __FILE__ ) );
 define( __NAMESPACE__ . '\PLUGIN_URL', \plugins_url( '/', __FILE__ ) );
+define( __NAMESPACE__ . '\OPTION_KEY', 'sft_feedback_page' );
 
+register_activation_hook( __FILE__, __NAMESPACE__ . '\activate' );
+register_deactivation_hook( __FILE__, __NAMESPACE__ . '\deactivate' );
 add_action( 'plugins_loaded', __NAMESPACE__ . '\load' );
 
+// Check if the page exists, and add it if not.
+add_action( 'init', __NAMESPACE__ . '\add_feedback_page' );
+
 /**
  * Include the rest of the plugin.
  */
 function load() {
 	require_once PLUGIN_DIR . 'includes/class-feedback.php';
 	require_once PLUGIN_DIR . 'includes/comment.php';
+	require_once PLUGIN_DIR . 'includes/form.php';
+	require_once PLUGIN_DIR . 'includes/page.php';
+}
+
+/**
+ * Create main Feedback page.
+ */
+function activate() {
+	add_feedback_page();
+}
+
+/**
+ * Create the Feedback page, save ID into an option.
+ */
+function add_feedback_page() {
+	$page_id = get_option( OPTION_KEY );
+	if ( $page_id ) {
+		return;
+	}
+
+	$organizer_note = '<!-- wp:paragraph {"textColor":"white","customBackgroundColor":"#94240b"} -->';
+	$organizer_note .= '<p style="background-color:#94240b" class="has-text-color has-background has-white-color">';
+	$organizer_note .= __( 'This page is a placeholder for the Speaker Feedback form. The content here will not be shown on the site.', 'wordcamporg' );
+	$organizer_note .= '</p>';
+	$organizer_note .= '<!-- /wp:paragraph -->';
+
+	$page_id = wp_insert_post( array(
+		'post_title'   => __( 'Leave Feedback', 'wordcamporg' ),
+		/* translators: Page slug for the feedback page. */
+		'post_name'    => __( 'feedback', 'wordcamporg' ),
+		'post_content' => $organizer_note,
+		'post_status'  => 'publish',
+		'post_type'    => 'page',
+	) );
+
+	if ( $page_id > 0 ) {
+		update_option( OPTION_KEY, $page_id );
+	}
+}
+
+/**
+ * Remove the feedback page.
+ */
+function deactivate() {
+	$page_id = get_option( OPTION_KEY );
+	wp_delete_post( $page_id, true );
+}
```

Comment:
Need to delete the option here as well, since the page is being permanently deleted.